ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/router_bgp.py

```diff
@@ -156,7 +156,7 @@ def _router_bgp_vrfs(self) -> list | None:
                     else:
                         target["route_targets"].append(vrf_rt)
 
-                for rt in vrf["additional_route_targets"]:
+                for rt in get(vrf, "additional_route_targets", default=[]):
```

Comment:
This never failed, so I suspect we insert the key somewhere in filtered tenants.